packages/ZbayLite/src/renderer/containers/widgets/createUsernameModal/CreateUsername.tsx

```diff
@@ -69,6 +69,9 @@ const CreateUsernameModal = () => {
     }
   }, [unregisteredCommunities])
   useEffect(() => {
+    if (!createUsernameModal.communityAction) {
```

Comment:
`initializedCommunitiesCount` and `allCommunitiesInitialized` was my doing and I can see that later Damian named his variable `initializedCommunities` which became very misleading. 
I still don't know what `unregisteredCommunitiesWithoutUserIdentity` are and I don't think that they are just 'initialized communities'.

Reply:
Also in `unregisteredCommunitiesWithoutUserIdentity` -> 'unregistered' and 'WithoutUserIdentity' are redundant so it's enough to name it `communitiesWithoutUserIdentity` to avoid confusion.